src/components/PatientBrowser.vue

```diff
@@ -371,7 +371,7 @@ export default {
       return '';
     },
 
-    ...mapActions(['selectBaseImage', 'updateScene', 'removeData']),
+    ...mapActions('visualization', ['selectBaseImage', 'updateScene', 'removeData']),
```

Comment:
We want to use the top-level `removeData` action, and that should call into the visualization model's removeData, so we should use the non-namespaced version of that action.

Reply:
Further evidence to clarify `removeData`!

Reply:
Agreed! We should rename other uses of `removeData` to be something more specific to that module.